packages/_private/angular-demo/src/app/app.component.css

```diff
@@ -0,0 +1,8 @@
+main {
+  width: 400px;
+  background: var(--syn-color-neutral-0);
+  border: 1px solid var(--syn-color-neutral-200);
+  border-radius: var(--syn-border-radius-small);
+  box-shadow: 0 0 25px 0 var(--syn-color-neutral-200);
+  padding: 10px;
+}
```

Comment:
Yes. However, its not as easy as I thought it would be to fit them into the Angular-App. Talk about being opinionated ;). https://github.com/SickDesignSystem/synergy/blob/feat/angular-wrapper/packages/_private/angular-demo/angular.json#L31